lib/ansible/executor/play_iterator.py

```diff
@@ -181,6 +181,8 @@ def __init__(self, inventory, play, play_context, variable_manager, all_vars, st
         }
         if gather_timeout:
             setup_task.args['gather_timeout'] = gather_timeout
+        if self._play.fact_path:
```

Comment:
@bcoca: Regarding your comment 
> a better way is to just have play default to the config setting

Do you mean the setting from `constants.py`?

If yes: `fact_path` is None by default, because the actual default comes from the setup module and I didn't want to duplicate that.